src/main/java/frc/robot/RobotContainer.java

```diff
@@ -12,15 +12,18 @@
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 import frc.robot.Constants.ShooterConstants;
 import frc.robot.Subsystems.Shooter;
+import frc.robot.Subsystems.TankDrive;
 import monologue.Logged;
 import monologue.Monologue;
 
 public class RobotContainer implements Logged {
     private final XboxController controller;
     private final Shooter shooter;
+    public final TankDrive tankdrive;
```

Comment:
Please do not make subsystems public.

Robot shouldn't know about the RobotContainer's subsystems. This removes the entire reason these were separated several years ago.

Use public methods on RobotContainer to duplicate the Robot test methods. Then call the container from the robot. In the container, make the calls to the subsystems.

public void testPeriodic() {
  m_robotContainer.testPeriodic();
}